viewportHax.js

```diff
@@ -28,3 +28,17 @@ window.addEventListener("scroll", (e) => {
 document.getElementById("chat_input")?.addEventListener("focus", (e) => {
   setTimeout(() => window.scrollTo(0, 0), 100);
 });
+
+if (isMobile) {
+  document.addEventListener(
+    "touchmove",
+    function (e) {
+      if (document.activeElement.id !== "chat_input") return;
+      const isChatView = e.target.closest("#view");
+      if (!isChatView) {
+        e.preventDefault();
+      }
+    },
+    { passive: false },
+  );
+}
```

Comment:
nit: maybe just leave a comment here saying that this is to prevent scroll everywhere but in `#view` cos we will probably forget at some point